startLuminoWithTransport.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+### Usage
+
+## This script assumes:
+
+# You have NVM installed
+# You have NVM 14.13.1 installed
+# You have ran npm install on the rif communications pub sub bootnode project
+# You have created the config files for the rif communications pub sub bootnode (i.e /config/nodeA and keys/nodeA)
+# You already know and configured the PORT in which rif comms node is going to run
+
+## Running
+
+# Run this script into the directory in which Lumino was git cloned
+# Specify the node configuration and the port that its already configured: i.e ./startLuminoWithTransport.sh nodeA 5013
+# The output log for rif comms node is on rif-communications-pub-sub-bootnode under the commsNodeLog.txt name
+
+RIF_COMMS_NODE_CONFIGURATION=$1
+RIF_COMMS_NODE_PORT=$2
+
+
+# move to the previous folder where is the rif comms node
+cd ..
```

Comment:
This can be replaced by something like this:
```bash
# Set node version 14.13.1
nvm use v14.13.1
npm run api-server --prefix ../rif-communications-pubsub-bootnode > commsNodeLog.log 2>&1

```

Reply:
done